### ISSUE TITLE
SEO

### ISSUE THREAD

**Issue by devdupont** [cforlando/cforlando.github.io#3]
codefororlando.com does not appear on the first page of Google search results ("Code For Orlando" & "codefororlando"), but everything else does no matter how obscure.

**Comment by Erin-Denton** [comment]
I noticed this too. I am not sure how to fix it. Right now the domain points to a wordpress site. When I log into the CMS there is a section for SEO but I am not familiar in SEO and am not sure what needs to be changed.

**Comment by devdupont** [comment]
Do we have a timeline for moving the url to the github.io site?

**Comment by Erin-Denton** [comment]
Not currently, I also need to work on the design with Chad, it might be faster to just find a nice jekyll template. I can do that :D

**Comment by akozlik** [comment]
I thought the plan was to adopt what Code for San Francisco did. Has that
changed?

**Comment by Erin-Denton** [comment]
That's right I remember we talked about just using another brigade's template. Lets do that https://github.com/sfbrigade/sfbrigade.github.io 

They use Jekyll and Bootstrap which is basically what kind of template I was going to look for. Problem Solved!

**Comment by akozlik** [comment]
Cool. I'm down with that too. Let's get it up and running and I can point
the DNS once it's done.

**Comment by chadmiller** [comment]
Here's what I made, and what cfsf made. They put a lot in "includes", which is complex for no good reason, IMO.

https://github.com/cforlando/cforlando.github.io/tree/master/_layouts

https://github.com/sfbrigade/sfbrigade.github.io/tree/master/_layouts
https://github.com/sfbrigade/sfbrigade.github.io/tree/master/_includes
